mmv1/templates/terraform/examples/resource_manager_capability.tf.tmpl

```diff
@@ -1,5 +1,5 @@
 resource "google_folder" "folder" {
-  provider         = google-beta
+  provider         = google
```

Comment:
Perhaps this line should be removed from the test so that the tests could run on both beta and GA.

Same for all the following lines with `provider = google`